kubetest/README.md

```diff
@@ -24,6 +24,12 @@ Note that developers frequently use `kubetest` by calling `go run hack/e2e.go`
 in the `kubernetes/kubernetes` repository. This `hack/e2e.go` program is a
 wrapper around updating `kubetest` (at most once a day) before calling it.
 
+If you're making a change to kubetest, watch the canary jobs after it has merged;
```

Comment:
you still need to push a new kubekins-e2e image which will update the latest tag, probably mention here as well, either sync with HEAD then push one yourself, or ask one of us who has permission to push it.